### PR TITLE
Add Python 3.10 support.

### PR DESCRIPTION
**Pull request by icemac**
Fix docs.
Fix badge.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,8 +2,12 @@
 zc.intid - Reduced-conflict integer id utility
 ==============================================
 
-.. image:: https://travis-ci.org/zopefoundation/zc.intid.svg?branch=master
-    :target: https://travis-ci.org/zopefoundation/zc.intid
+.. image:: https://github.com/zopefoundation/zc.intid/actions/workflows/tests.yml/badge.svg
+    :target: https://github.com/zopefoundation/zc.intid/actions/workflows/tests.yml
+
+.. image:: https://readthedocs.org/projects/zcintid/badge/?version=latest
+        :target: http://zcintid.readthedocs.io/en/latest/?badge=latest
```
Comment (perrinjerome) [inline README.rst]: shouldn't this be a http**s** url ? (also I'm not sure the `?badge=latest` is needed )

Comment (perrinjerome) [reply README.rst]: (ah sorry this code is just moved around, but maybe this is something that can be fixed as well ;) )

**Review by perrinjerome — APPROVED** [review]
I don't see anything bad except the small details I noted.

I don't know this well, so I don't know if it's OK for me to click Approve (but I clicked anyway)

**Review by jamadden — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (icemac) [reply README.rst]: Fixed, thank yo for brining it up.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**